app/assets/stylesheets/config/_flexbox.scss

```diff
@@ -70,6 +70,12 @@
   align-items: center;
 }
 
+.flex-row-center-centered {
```

Comment:
You couldn't possibly have known this, but we already have a class named 'centered-item' that does exactly this

Reply:
as we discused i replaced every centered-item by flex-row-center-centered